### DISCUSSION TITLE
How to hide literal value at logs

### DISCUSSION THREAD

**Discussion by EizFeren** [sequelize/sequelize#15573] [Help & Questions] [answered]
Hi, I want to know, is there a way at sequelize to hide literal for logging?

My case - I'm using decimal type at postgres for storing balance, so I need to use strings at JS (code sample below):

```js
bank.update({
  balance: this.sequelize.literal(`${bank.balance} + ${amount}`),
});
```

And there is sequelize logs output:

```
UPDATE "banks" SET "balance"=456.762 + -100.019,"updatedAt"=$1 WHERE "id" = $2
```

Can I somehow to hide this `456.762 + -100.019`, same as `$1` and `$2`?

**Comment by ephys** [comment, marked as answer]
I would normally tell you to use bind parameters, but I think that feature seems to be missing from the update method? If so that's something we should fix

This is what it would look like if it's supported, you can always give it a try:

```ts
bank.update({
  balance: this.sequelize.literal(`$balance + $amount`),
}, {
  bind: {
    balance: bank.balance,
    amount,
  },
});
```

If that doesn't work, we'll open a feature request.

**Reply by EizFeren** [reply]
Yes, I cant use bind for model update, there's Typescript error - `Object literal may only specify known properties, and 'bind' does not exist in type 'InstanceUpdateOptions<any>'`

**Reply by ephys** [reply]
I just looked into this, and I remember now

I implemented that feature in Sequelize 7, so this is unfortunately not available in Sequelize 6 and prior (it required introducing a small breaking change), but the typing is still incorrect. I'll have to open a PR for that

Unfortunately that means that until you update to Sequelize 7 (which is currently in alpha), the only way to use bind parameters here is to use a raw query using `sequelize.query` :(

Edit: you can also disable the logging of queries

**Reply by EizFeren** [reply]
Anyway thank you for help, I'll inevstigate sequelize 7 more when it will be main
